### PR TITLE
Added extensions to the graphql request

### PR DESCRIPTION
**Pull request by michaelstaib**
Hi Ivan, 

I have added the extensions property to the request.

Also shouldn't we describe how the response should look like?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,12 +88,15 @@ A request for execution should contain the following request parameters:
 * `query` - A Document containing GraphQL Operations and Fragments to execute.
 * `operationName` - [*Optional*]: The name of the Operation in the Document to execute.
 * `variables`- [*Optional*]: Values for any Variables defined by the Operation.
+* `extensions`- [*Optional*]: This entry is reserved for implementors to extend the protocol however they see fit.
```
Comment (sungam3r) [inline README.md]:
```suggestion
* `extensions` - [*Optional*]: This entry is reserved for implementors to extend the protocol however they see fit.
```

Comment (sungam3r) [reply README.md]: Also the line above.

**Comment by binaryseed** [comment]
Is there any consensus on this one? It makes sense to me since it mirrors usage of `extensions` elsewhere, like the response body

File: README.md [hunk 2]
```diff
@@ -88,12 +88,15 @@ A request for execution should contain the following request parameters:
 * `query` - A Document containing GraphQL Operations and Fragments to execute.
 * `operationName` - [*Optional*]: The name of the Operation in the Document to execute.
 * `variables`- [*Optional*]: Values for any Variables defined by the Operation.
```
Comment (sungam3r) [inline README.md:90]:
```suggestion
* `variables` - [*Optional*]: Values for any Variables defined by the Operation.
```

**Comment by michaelstaib** [comment]
@IvanGoncharov I agree with that since otherwise, `extensions` would be pointless to have.

**Comment by michaelstaib** [comment]
@IvanGoncharov what about `id`. Which is used by Facebook and also by Apollo for persisted queries? Just looked at the spec. It could go into extensions but until now it is a top-level field and would break the current requests.

**Comment by IvanGoncharov** [comment]
@michaelstaib On the last WG, we discussed that since we will require `application/graphql+json` we can make a clean start and allow servers to distinguish pre and post spec requests.

**Comment by sjparsons** [comment]
Can we merge this? It makes sense to me, but I wasn't clear @IvanGoncharov if your comments "I will forbid all other top-level properties" meant that you now do not approve of this PR?

**Comment by michaelstaib** [comment]
This was meant regarding things like id.

**Pull request merged**